Vagrantfile

```diff
@@ -382,8 +382,8 @@ Vagrant.configure('2') do |config|
 
   # Configurations from 1.0.x can be placed in Vagrant 1.1.x specs like the following.
   config.vm.provider :virtualbox do |v|
-    # Move the ubuntu-bionic-18.04-cloudimg-console.log file to log directory.
-    v.customize ['modifyvm', :id, '--uartmode1', 'file', File.join(vagrant_dir, 'log/ubuntu-bionic-18.04-cloudimg-console.log')]
+    # Move the ubuntu-focal-20.04-cloudimg-console.log file to log directory.
+    v.customize ['modifyvm', :id, '--uartmode1', 'file', File.join(vagrant_dir, 'log/ubuntu-focal-20.04-cloudimg-console.log')]
```

Comment:
we don't use this file, it'd be great if we could eliminate this completely

Reply:
done

Reply:
I'm not sure that removes the file though, just puts it in the root folder

Reply:
We should change thh uartmode from `file` to `disconnected` https://github.com/ansible-community/molecule/pull/1600/files

Reply:
This makes sense. Done